wallets/metamask/test/e2e/connectToDapp.spec.ts

```diff
@@ -1,9 +1,9 @@
-import { testWithSynpress } from '@synthetixio/synpress-fixtures'
-import { MetaMask, unlockForFixture } from '../../../src'
+import { testWithSynpress } from '@synthetixio/synpress-test'
```

Comment:
I'm not sure if `synpress-test` is the best name here for this package.. something like `synpress-core` sounds better, but it's already taken afaik

Reply:
We can more it to the `synpress-core`. At first I expected more files in that `...-test` package but now I am fine with a different location.

Reply:
rename `@synthetixio/synpress-test` to `@synthetixio/synpress`